src/worldcereal/openeo/mapping.py

```diff
@@ -155,7 +159,10 @@ def _croptype_map(
     # Postprocess
     if postprocess_parameters.enable:
         if postprocess_parameters.save_intermediate:
-            classes = classes.save_result(format="GTiff")
+            product_type = WorldCerealProductType.CROPTYPE.value
+            classes = classes.save_result(
+                format="GTiff", options=dict(filename_prefix=f"{product_type}-raw")
+            )
         # TODO: check how 254 value is treated during postprocessing
```

Comment:
Tis TODO is there for a while. You've been looking into postprocessing already. Could you check for this comment and once verified remove the TODO?